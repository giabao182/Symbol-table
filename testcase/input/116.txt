INSERT a number
INSERT b number
ASSIGN a 10
BEGIN
INSERT c string
ASSIGN c 'hi'
BEGIN
ASSIGN b 20
ASSIGN d 30
END
END