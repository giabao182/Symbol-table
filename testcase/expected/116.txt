success
success
success
success
success
success
Undeclared: ASSIGN d 30